### DISCUSSION TITLE
Feature Request - Cloud Template Library

### DISCUSSION THREAD

**Discussion by sflwa** [elementor/elementor#28816] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I have "templates" that I like to use on multiple sites - the issue is that I have to have make sure to keep a JSON file on my machine and update it from time to time - and  if I have multiple layouts I'd like to use that requires multiple files and I may not want to upload them all to every site

### Describe the solution you'd like.

I'd like to ability to store my templates via Elementor in the cloud to have them accessible across multiple websites

### Describe alternatives you've considered.

I keep local JSON files in a folder

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by CaptainSeebas** [comment]
Same here! I removed my -exact same- post ([29110](https://github.com/orgs/elementor/discussions/29110)).
Glad there are others with the same request, hopefully the team will listen!
